osidb/models.py

```diff
@@ -1318,7 +1255,7 @@ def get_queryset(self):
         return super().get_queryset()
 
 
-class FlawMeta(AlertMixin, TrackingMixin):
+class FlawMeta(AlertMixin, TrackingMixin, ACLMixin):
```

Comment:
`FlawMeta` inherits from `ACLMixin` but `FlawMetaManager` is based on default `models.Manager` instead of `ACLMixinManager` and thus it lacks some embargoed annotation for quering based on the embargoed status.

Reply:
makes sense, added `ACLMixinManager` to FlawMetaManager heritage.